static/js/add-entry.js

```diff
@@ -0,0 +1,46 @@
+$('.add-entry').on('submit', function(e) {
+    // Disable default behavior
+    e.preventDefault();
+    // Grab the title and content of our post
+    var title = $('#title').val();
+    var text = $('#text').val();
+    var split_path = window.location.pathname.split("/");
```

Comment:
should be possible to put the id of the post you are editing into the form as a hidden field, thus removing the need to scrape the URL to find this data.